app/main/views.py

```diff
@@ -163,7 +163,9 @@ def index():
 @main.route('/standings/')
 def standings():
     years = sorted(set(
-        int(date.year) for (date,) in Race.query.with_entities(Race.date).all()),
+        int(date.year) for (date,) in Race.query.with_entities(Race.date)\
```

Comment:
Style nit: the backslashes are superfluous.  PEP-8 checkers complain about this.